### PR TITLE
Added UI currency support

### PR DESCRIPTION
**Pull request by Danstahr**
The app is currently USD-only. While many users don't require multiple currencies it's useful for them to switch the UI to a different currency. 

This pull request adds the possibility to choose a different currency to be displayed in the UI.

### CODE REVIEW

File: src/main/resources/migrations/03-users-add-currency.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="3" author="danstahr">
+        <addColumn tableName="users">
+            <column name="currency" type="varchar(10)" />
+        </addColumn>
+    </changeSet>
+</databaseChangeLog>
```
Comment (paukiatwee) [inline src/main/resources/migrations/03-users-add-currency.xml]: Please end with newline

File: src/main/resources/app/partials/profile.html [hunk 2]
```diff
@@ -31,9 +31,18 @@ <h3 class="box-title"></h3>
...
-                  </span>
+                      <span class="help-block" ng-show="form.name.$invalid && form.name.$dirty">
+                        {{errorMessage('name')}}
+                      </span>
+                    </div>
+                  </div>
+                  <div class="form-group has-feedback" ng-class="errorClass('currency')">
+                    <label for="currency" class="col-sm-3 control-label">Currency Symbol</label>
+                    <div class="col-sm-7">
+                      <input type="text" class="form-control" name="currency" id="currency" placeholder="Currency Symbol" ng-model="user.currency">
```
Comment (paukiatwee) [inline src/main/resources/app/partials/profile.html]: Please use **Enter Currency Symbol** for consistency with other placeholders.

File: src/main/resources/migrations/03-users-add-currency.xml (hunk 1 above)
Comment (Danstahr) [inline src/main/resources/migrations/03-users-add-currency.xml]: Done.

File: src/main/resources/app/partials/profile.html (hunk 2 above)
Comment (Danstahr) [inline src/main/resources/app/partials/profile.html]: Done.

**Pull request merged**